src/Common/Admin/Help_Hub/Hub.php

```diff
@@ -0,0 +1,537 @@
+						'sysinfo_error_code_text'    => _x( 'Code:', 'Error code label for system info optin', 'tribe-common' ),
+						'sysinfo_error_status_text'  => _x( 'Status:', 'Error status label for system info optin', 'tribe-common' ),
+					],
+				],
+			]
+		);
+
+		wp_enqueue_script( 'jquery-ui-accordion' );
+	}
+
+	/**
+	 * Generates a telemetry opt-in link.
+	 *
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+	public static function get_telemetry_opt_in_link(): string {
+		return add_query_arg(
+			[
+				'page'      => 'tec-events-settings',
+				'tab'       => 'general-debugging-tab',
+				'post_type' => 'tribe_events',
+			],
+			admin_url( 'edit.php' )
+		);
+	}
+
+	/**
+	 * Generates and outputs iframe content when appropriate.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 * @throws RuntimeException If data has not been set using setup.
+	 */
+	public function generate_iframe_content(): void {
+		$this->ensure_data_is_set();
+		$page   = tribe_get_request_var( 'page' );
+		$iframe = tribe_get_request_var( 'embedded_content' );
+
+		if ( empty( $page ) || 'tec-events-help-hub' !== $page || empty( $iframe ) ) {
+			return;
+		}
+
+		/**
+		 * Fires before the Help Hub iframe content is rendered.
+		 *
+		 * Use this hook to enqueue additional assets, modify iframe-specific content,
+		 * or take other actions just before the iframe content is generated.
+		 *
+		 * @since TBD
+		 *
+		 * @param Hub $this The Hub instance.
+		 */
+		do_action( 'tec_help_hub_before_iframe_render', $this );
+
+		add_action( 'wp_enqueue_scripts', [ $this, 'enqueue_help_page_iframe_assets' ] );
+		// phpcs:ignore WordPressVIPMinimum.UserExperience.AdminBarRemoval.RemovalDetected
+		show_admin_bar( false );
+		$this->render_template( 'help-hub/support/iframe-content' );
+
+		/**
+		 * Fires after the Help Hub iframe content has been rendered.
+		 *
+		 * Use this hook to perform actions or cleanup tasks after the iframe content
+		 * has been generated and displayed.
+		 *
+		 * @since TBD
+		 *
+		 * @param Hub $this The Hub instance.
+		 */
+		do_action( 'tec_help_hub_after_iframe_render', $this );
+
+		exit;
+	}
+
+	/**
+	 * Enqueues assets specific to the iframe content and removes theme styles.
+	 *
+	 * @since TBD
+	 * @return void
+	 */
+	public function enqueue_help_page_iframe_assets(): void {
+		define( 'IFRAME_REQUEST', true );
+
+		tribe_asset(
+			Tribe__Main::instance(),
+			'help-hub-iframe-style',
+			'help-hub-iframe.css',
+			null,
+			'wp_enqueue_scripts'
+		);
+
+		tribe_asset(
+			Tribe__Main::instance(),
+			'help-hub-iframe-js',
+			'admin/help-hub-iframe.js',
+			null,
+			'wp_enqueue_scripts',
+			[
+				'localize' => [
+					'name' => 'helpHubSettings',
+					'data' => [
+						'docsbot_key'    => $this->config->get( 'TEC_HELP_HUB_CHAT_DOCSBOT_SUPPORT_KEY' ),
+						'zendeskChatKey' => $this->config->get( 'TEC_HELP_HUB_CHAT_ZENDESK_CHAT_KEY' ),
+					],
+				],
+			]
+		);
+
+		self::dequeue_theme_styles();
```

Comment:
Is there any reason to do this here, rather than via a separate `wp_enqueue_scripts` hook above?
It's kinda hard to find if we do it this way.

There are several hooks that get added in different places - can any of them get moved to `register_hooks()`?

Reply:
That makes sense. I created a new method `register_iframe_hooks`.